### PR TITLE
Avoid building unused package dependencies

### PR DESCRIPTION
**Pull request by borkra**
This prevents building of unused packages during OpenWRT image build

### CODE REVIEW

File: net/gensio/Makefile [hunk 1]
```diff
@@ -52,6 +52,7 @@ CONFIGURE_VARS += \
 	ac_cv_lib_pam_pam_start=$(if $(CONFIG_GENSIO_SSHD),yes,no)
 
 ifneq ($(CONFIG_PACKAGE_python3-gensio),)
+
```
Comment (cotequeiroz) [inline net/gensio/Makefile:55]: We should not add this blank line here.

**Review by cotequeiroz — CHANGES_REQUESTED** [review]

File: net/gensio/Makefile [hunk 2]
```diff
@@ -130,8 +131,8 @@ $(call Package/gensio/Default)
   TITLE+= (Python3-bindings)
   SECTION:=lang
   CATEGORY:=Languages
-  BUILD_DEPENDS:=+swig +python3
-  DEPENDS:=+python3-light +libgensio
+  BUILD_DEPENDS:=+PACKAGE_python3-gensio:swig +PACKAGE_python3-gensio:python3
```
[the remark this answers is not in the document]
Comment (borkra) [reply net/gensio/Makefile:134]: It seems like BUILD_DEPENDS is there until I changed it, it kept building Python. But I will look more.

File: net/gensio/Makefile (hunk 1 above)
Comment (borkra) [reply net/gensio/Makefile]: Fixed

File: net/gensio/Makefile (hunk 2 above)
[the remark this answers is not in the document]
Comment (borkra) [reply net/gensio/Makefile]: Fixed

**Comment by borkra** [comment]
@cotequeiroz could you please remove 'changes requested' since they were done

**Review by cotequeiroz — APPROVED** [review]
I’ll let the Ci build it, then it can be merged.

**Pull request merged**